src/auditlog/mixins.py

```diff
@@ -1,6 +1,12 @@
 import json
 
 from django.conf import settings
+
+try:
+    from django.utils.safestring import mark_safe
+except ImportError:
```

Comment:
Why is this in a try/except? as far as I can tell `django.utils.safestring.mark_safe` is in Django 1.8, 1.9, 1.10, 1.11, and 2.0 with no plans for deprecation.